src/components/list-item.js

```diff
@@ -11,11 +12,13 @@ export default class ListItem extends Component {
     return (
       <StyledListItem
         className={this.props.className}
+        bulletStyle={this.props.bulletStyle}
         styles={[
           this.context.styles.components.listItem,
           getStyles.call(this),
           typefaceStyle,
-          this.props.style
+          this.props.style,
+          getBulletStyle(this.props.bulletStyle, true)
```

Comment:
Adding a check for the prop `bulletStyle` here might solve some of the snapshot issues where an empty array is being passed into the `styles` obj